designer/client/conditions/inline-conditions.js

```diff
@@ -77,6 +73,15 @@ class InlineConditions extends React.Component {
     }
   }
 
+  onClickSave = async () => {
+    const copy = this.props.data.clone()
+    const conditionResult = await InlineConditionHelpers.storeConditionIfNecessary(copy, this.state.conditions)
+    await this.props.data.save(conditionResult.data)
```

Comment:
destructure your props/state for brevity. especially data since you use it a few times. `const { data } = this.props`